src/Constants/Flags.hs

```diff
@@ -0,0 +1,32 @@
+module Constants.Flags where
```

Comment:
We can debate but I don't think it's all constants or that it is constants at all actually. It's just types and I would rather colocate them with the modules using them. Eventually adding functions that actually transform flags in there. I don't feel strongly though.

Reply:
Yeah, I think it started out as constants but they gave way to types — happy to migrate if you have an opinion, otherwise I can make a suggestion like: `Tools/CommandLineFlags.hs` or similar?